lib/main.js

```diff
@@ -115,6 +115,8 @@ const Hydrogen = {
           this.handleKernelCommand({ command: "interrupt-kernel" }),
         "hydrogen:restart-kernel": () =>
           this.handleKernelCommand({ command: "restart-kernel" }),
+        "hydrogen:refresh-all-result-bubbles": () =>
```

Comment:
I think `restart-kernel-and-re-evaluate-bubbles` or `refresh-all-result-bubbles` are fine. I'll probably just start typing "bubbles"

Whatever we go with we can just explain the specifics in the docs 📖